### PR TITLE
[perceval] Update license and copyright information

### PR DESCRIPTION
**Pull request by vchrombie**
This PR updates license information for the remaining source code files in Perceval. 
This is an extension of #623.

I tried updating this using the tool, [vchrombie/grimoirelab-scripts](https://github.com/vchrombie/grimoirelab-scripts).

### CODE REVIEW

File: tests/test_pipermail.py [hunk 1]
```diff
@@ -17,6 +18,10 @@
 #
 # Authors:
 #     Santiago Dueñas <sduenas@bitergia.com>
+#     Stephan Barth <stephan.barth@gmail.com>
+#     Valerio Cosentino <valcos@bitergia.com>
+#     Miguel Ángel Fernández <mafesan@bitergia.com>
+#     Harshal Mittal <harshalmittal4@gmail.com>
```
Comment (vchrombie) [inline tests/test_pipermail.py]: I checked the test_pipermail.py commit history and I see zhquan as one of the author, but the list doesn't have his name.
https://github.com/chaoss/grimoirelab-perceval/commit/fca976a0a0e5dbfc6b895da986a502030740f22c

Also, for reference [github api commits - test_pipermail.py](https://api.github.com/repos/chaoss/grimoirelab-perceval/commits?path=tests/test_pipermail.py)

Did I assume anything wrong over here?

Comment (valeriocos) [reply tests/test_pipermail.py]: I have to check the tests for pipermail. Tomorrow I'll have a look and let you know. Thanks for raising this question!

**Comment by valeriocos** [comment]
can you update the commit description by adding some info about the files modified (e.g., tests, bin and setup.py)? Thanks

**Comment by vchrombie** [comment]
> Please, consider to submit also PRs to ELK and mordred, and the other components :)

Sure @valeriocos.

**Comment by vchrombie** [comment]
I just had one doubt in this PR. https://github.com/chaoss/grimoirelab-perceval/pull/624#discussion_r389400933

The rest all looks good to me. 

@valeriocos, WDYT?

File: tests/test_pipermail.py (hunk 1 above)
Comment (valeriocos) [reply tests/test_pipermail.py]: sorry for the late reply! ok, I understand that he should be added.

Comment (vchrombie) [reply tests/test_pipermail.py]: Thanks for the reply.

I'll update the PR and let you know soon. :slightly_smiling_face:

**Comment by valeriocos** [comment]
Please @vchrombie ping me when the PR is ready for review, thanks!

**Comment by vchrombie** [comment]
Hi @valeriocos 

I updated the PR and I think it is in good shape now. The Travis CI build also passed. :relieved: 
You can review it now.

**Pull request merged**